pkg/console/subresource/route/route.go

```diff
@@ -242,6 +241,9 @@ func certificateVerifier(customCert []byte) error {
 	if now.Before(certificate.NotBefore) {
 		return fmt.Errorf("custom TLS certificate is not valid yet")
 	}
+	if !oscrypto.CertHasSAN(certificate) {
```

Comment:
`certificateVerifier` has no godoc, I am therefore assuming that it can be used generically on any CA and server certs. CA certificates won't contain SANs, though, so you may want to move this somewhere outside, or rename `certificateVerifier` to `serverCertificateVerifier`, or create a wrapping `serverCertificateVerifier`.

Reply:
created a wrapper function

Reply:
it should be wrapped the other way around, this way you're still checking the cert for SAN while the certificate can be a CA cert.